code/game/mecha/mecha.dm

```diff
@@ -1561,7 +1561,7 @@
 		var/mob/occupant = P.occupant
 
 		user.visible_message("<span class='notice'>\The [user] begins opening the hatch on \the [P]...</span>", "<span class='notice'>You begin opening the hatch on \the [P]...</span>")
-		if (!do_after(user, 40, needhand=0))
```

Comment:
we've never used global.thing before, i don't see any reason to start doing it now

Reply:
the do_after proc is overwritten for this machine.